hack/jenkins/upload-to-gcs.sh

```diff
@@ -115,11 +121,26 @@ function upload_artifacts_and_build_result() {
       echo "Uploading build log"
       gsutil -q cp -Z -a "${gcs_acl}" "${WORKSPACE}/build-log.txt" "${gcs_build_path}"
     fi
+
+    # For pull jobs, keep a canonical ordering for tools that want to examine
+    # the output.
+    echo "aoeu debug: ${gcs_indirect_path}"
```

Comment:
How can I test my changes then?

On Fri, Jun 24, 2016 at 9:26 AM, Jeff Grafton <notifications@github.com>
wrote:

> In hack/jenkins/upload-to-gcs.sh
> <https://github.com/kubernetes/kubernetes/pull/27898#discussion_r68423708>
> :
>
> > @@ -115,11 +121,26 @@ function upload_artifacts_and_build_result() {
> >        echo "Uploading build log"
> >        gsutil -q cp -Z -a "${gcs_acl}" "${WORKSPACE}/build-log.txt" "${gcs_build_path}"
> >      fi
> > +
> > +    # For pull jobs, keep a canonical ordering for tools that want to examine
> > +    # the output.
> > +    echo "aoeu debug: ${gcs_indirect_path}"
>
> yeah, it always downloads upload-to-gcs.sh from HEAD:
> https://github.com/kubernetes/test-infra/blob/b12293e6c4a49a968705775b06656d452f6db9e0/jenkins/job-configs/kubernetes-jenkins-pull/global.yaml#L7-L13
>
> I think we discussed changing this to download only if necessary. cc
> @spxtr <https://github.com/spxtr>
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/27898/files/3f235d57378a8405d3fc05867aebd172af9b4980#r68423708>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAnglv3AGA4EXUeOlUohlb1Q8g8CudM1ks5qPAU3gaJpZM4I8Noc>
> .
>